parser/src/aspects/function_declaration.rs

```diff
@@ -327,7 +301,7 @@ mod tests {
         assert_eq!(
             errs,
             vec![ParseError {
-                message: "expected start of parameter list".to_string(),
+                message: "expected start of list expression".to_string(),
```

Comment:
```suggestion
                message: "expected start of parameter list".to_string(),
```
It is not an expression.